dask/dataframe/utils.py

```diff
@@ -297,7 +299,9 @@ def make_meta_index(x, index=None):
     return x[0:0]
 
 
-@make_meta.register(object)
+@make_meta_obj.register(
+    (pd.Series, pd.DataFrame, pd.Index, pd.MultiIndex, sp.csr.csr_matrix)
```

Comment:
It seems like adding a `scipy` dependency makes this PR a bit "heavier" than it needs to be.  Maybe we could just do something like:

```python
object_types = (pd.Series, pd.DataFrame, pd.Index, pd.MultiIndex)
try:
    import scipy.sparse as sp

    object_types += (sp.csr.csr_matrix,)
except ImportError:
    pass
```
.. and then use this tuple for `make_meta_obj.register`?

Reply:
Done ! 👍